configure.ac

```diff
@@ -2000,6 +2000,19 @@ then
 fi
 
 
+AC_ARG_ENABLE([no-neon],
```

Comment:
Once PR #5152 is merged I'd like you to use the existing pattern for detecting neon that Sean put together in `--enable-armasm`.

Reply:
I need you to explain this comment.

I have Sean's configure.ac file up now.  I don't see the check for NEON.

Reply:
@llcodesmith , please rebase this to latest master and use the neon detection Sean implemented. You should be able to eliminate this new `no-neon`.